### ISSUE TITLE
Feature - Applicant Profile

### ISSUE THREAD

**Issue by bitliege** [GCTC-NTGC/TalentCloud#4012]
# Description
In order to accommodate the new application process, the applicant profile area has to be rebuilt to reflect the new tool.

Right now, the profile contains:
- basic info
- education
- skills
- references
- work samples

We need it to contain:
- basic info (right now we're assuming this page can stay the same, waiting on @gobyrne to confirm)
- experience (the same UI from the application, however *without* the ability to ADD skills)

This will also require notification intervention (speak with @JasmitaChawla) so the users understand how their existing data will be handled, including what will be reused, what will be deleted, and what will be saved for potential use at a later date.

# Images
https://xd.adobe.com/view/4ac43ee7-a919-4397-6888-ce98019603f7-3cf5/screen/66c2f58a-92f1-468d-8268-921b82d07e2c/

- [ ] This task requires a unit test.
- [ ] This task is complex and needs acceptance criteria.

# Acceptance criteria
- [ ] make sure that the content from applications is reflected in this UI and vice versa
- [ ] the mockup is missing UI for a sidebar, but it already exists in the current template and should remain
 - the sidebar should contain links to:
  - my basic info
  - my experience
  - my account settings

**Comment by gobyrne** [comment]
Duplicate of #4126
